index.js

```diff
@@ -69,6 +69,13 @@ function getTextBetweenHighlightBoundaries({text, OPEN_TAG, CLOSE_TAG}) {
 
 function trimTextUntilSize(text, maxLengthAround, omission) {
   const wordsInput = text.split(' ').reverse();
+  const firstWord = [...wordsInput[0]].reverse().join('');
```

Comment:
Why spread array into array? It will be the same

Reply:
I'm spreading first item of array, which is string.